test/pylib/report_plugin.py

```diff
@@ -19,18 +19,25 @@ def pytest_runtest_makereport(self):
         report = outcome.get_result()
         report.nodeid = f"{report.nodeid}.{self.mode}.{self.run_id}"
 
-    @pytest.fixture(scope="function", autouse=True)
-    def allure_set_mode(self, request, mode):
```

Comment:
To me it looks like you applied fixes to review comments in the wrong commit. This particular diff should be applied to the first commit (Enable allure for python test) and not the second one (Clean duplicated arg for test suite).

Reply:
Yes, I've messed up commits.